pkg/kubelet/kubelet.go

```diff
@@ -1183,8 +1183,7 @@ func (kl *Kubelet) initializeModules() error {
 	// Step 5: Start container manager.
 	node, err := kl.getNodeAnyWay()
 	if err != nil {
-		glog.Errorf("Cannot get Node info: %v", err)
```

Comment:
return fmt.Errorf("Kubelet failed to get node info.")----After the return, this will be recorded,, together with the record of it; the other parts of the function is also the way